src/dso_api/dynamic_api/fields.py

```diff
@@ -91,48 +89,14 @@ def to_representation(self, value):
         return f"{value}?{sas_token}"
 
 
-class LooseRelationUrlField(serializers.CharField):
-    """A url field for very loose relations to temporal datasets
-    The relation has to be defined as: "<dataset>:<table>:<column>" in the schema
-    The specific definition of the column signals that the relation is
-    very loose and a url is constructed without any checking.
-    """
-
-    def to_representation(self, value):
-        from .urls import app_name
-
-        request = self.context["request"]
-        view = self.context["view"]
-        relation = view.model._meta.get_field(to_snake_case(self.field_name)).relation
-        dataset_name, table_name = (to_snake_case(part) for part in relation.split(":"))
-        # We force that the incoming value is interpreted as the
-        # pk, although this is not always the 'real' pk, e.g. for temporal relations
-        kwargs = {"pk": value}
-        return reverse(
-            f"{app_name}:{dataset_name}-{table_name}-detail",
-            kwargs=kwargs,
-            request=request,
-        )
-
-
-class HALLooseRelationUrlField(LooseRelationUrlField):
+class HALLooseRelationUrlField(HyperlinkedRelatedField):
     """Wrap the URL from LooseRelationUrlField according to HAL specs."""
 
-    def to_representation(self, value):
-        href = super().to_representation(value)
-        view = self.context["view"]
-        field = view.model._meta.get_field(to_snake_case(self.field_name))
-        relation = field.relation
-        dataset_name, table_name = (to_snake_case(part) for part in relation.split(":"))
-        result = {"href": href}
-
-        if view.model.has_display_field():
-            result["title"] = str(value)
-
-        related_identifier = first(field.related_model.table_schema().identifier)
-        result[related_identifier] = value
-        return result
-
+    def use_pk_only_optimization(self):
+        return True
 
-class LooseRelationUrlListField(serializers.ListField):
```

Comment:
Ive removed this class since `ModelSerializer`s can figure out that this needs to be instantiated with `many=True` in case of an mn rel